public/js/app/pages/CreateNewMemo.jsx

```diff
@@ -214,13 +232,13 @@ function CreateNewMemo(props) {
       window.location = continueToEditorUrl
     } else if (sortedApplicationCodes.length > 0 && !existingDraftEndPoint) {
       const courseTitle = combinedCourseName(courseCode, course, memoCommonLangAbbr)
-
       // Create new draft from chosen semester rounds
       const body = {
         courseCode,
         courseTitle,
         memoName: newMemoName,
         memoCommonLangAbbr,
+        ladokRoundIds: _getSortedCourseRoundsIds(semester, sortedApplicationCodes, lastTerms),
```

Comment:
When application codes are added, then we dont need anymore `ladokRoundIds` so you can delete all code connected with it